### PR TITLE
WIP: Add seaborn

### PR DESCRIPTION
**Pull request by r-jain1**
Add seaborn, a statistical data visualization library

### CODE REVIEW

File: recipes/seaborn/meta.yaml [hunk 1]
```diff
@@ -12,13 +12,14 @@ source:
...
   number: 0
-  script: python setup.py install
+  script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - numpy
     - matplotlib
+    -setuptools
```
Comment (patricksnape) [inline recipes/seaborn/meta.yaml]: Missing space

File: recipes/alabaster/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    Alabaster is a visually clean, responsive, and configurable Sphinx
+    theme. It is sidebar-enabled and easy to install/use as a Python
+    package. As of Sphinx 1.3, Alabaster is selected as the default
+    theme for Sphinx.
+  doc_url: https://anaconda.org/anaconda/alabaster
+  dev_url: https://github.com/bitprophet/alabaster
+
+extra:
+  recipe-maintainers:
+    -msarahan
```
Comment (patricksnape) [inline recipes/alabaster/meta.yaml]: Spaces missing for these two as well.

File: recipes/seaborn/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  summary: Statistical data visualization
+  description: |
+    Seaborn is a Python visualization library based on matplotlib. It
+    provides a high-level interface for drawing attractive statistical graphics.
+  doc_url: http://stanford.edu/~mwaskom/software/seaborn/
+  dev_url: https://github.com/mwaskom/seaborn
+
+extra:
+  recipe-maintainers:
+    -msarahan
```
Comment (patricksnape) [inline recipes/seaborn/meta.yaml]: Space missing here too

File: recipes/seaborn/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  build:
+    - python
+    - numpy
+    - matplotlib
+    - setuptools
+  run:
+    - python
+    - numpy
```
Comment (patricksnape) [inline recipes/seaborn/meta.yaml]: @jakirkham I guess we expect this to be `numpy x.x`?

Comment (jakirkham) [inline recipes/seaborn/meta.yaml]: Only if they are linking to it, which I don't think is the case with `seaborn` as it is pure Python.

File: recipes/seaborn/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  description: |
+    Seaborn is a Python visualization library based on matplotlib. It
+    provides a high-level interface for drawing attractive statistical graphics.
+  doc_url: http://stanford.edu/~mwaskom/software/seaborn/
+  dev_url: https://github.com/mwaskom/seaborn
+
+extra:
+  recipe-maintainers:
+    -msarahan
+    -r-jain1
```
Comment (jakirkham) [inline recipes/seaborn/meta.yaml]: Also, space needed here.

File: recipes/alabaster/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
+{% set version = "0.7.7" %}
+
+package:
+  name: alabaster
+  version: {{version}}
+
+source:
+  fn: alabaster-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/source/a/alabaster/alabaster-{{version}}.tar.gz
```
Comment (jakirkham) [inline recipes/alabaster/meta.yaml]: Could we please switch to pypi.io? It should save you time later.

Also, going from `{{version}}` to `{{ version }}` would be nice for readability.

File: recipes/alabaster/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - alabaster
+  commands:
+    - python -c "import alabaster; print(alabaster.get_path())"
+
+about:
+  home: http://www.sphinx-doc.org/en/stable/index.html
+  license: BSD
```
Comment (jakirkham) [inline recipes/alabaster/meta.yaml]: Please add the number of clauses.

File: recipes/seaborn/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - make test
+
+about:
+  home: http://stanford.edu/~mwaskom/software/seaborn/
+  license: BSD
```
Comment (jakirkham) [inline recipes/seaborn/meta.yaml]: Please add the number of clauses.

File: recipes/seaborn/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,50 @@
+{% set version = "0.7.0" %}
```
Comment (pmlandwehr) [inline recipes/seaborn/meta.yaml]: Seaborn's now at 0.7.1

**Comment by jakirkham** [comment]
Anything we can help you with here, @r-jain1?

**Comment by r-jain1** [comment]
@jakirkham Sorry I'll fix this up soon, but thanks I'll let you know if I need help!

**Comment by jakirkham** [comment]
No worries. Thanks @r-jain1. 😄

File: recipes/seaborn/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+    - numpy
+    - matplotlib
+    - scipy
+    - pandas
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - make test
```
Comment (jakirkham) [inline recipes/seaborn/meta.yaml]: I think this is unknown. 😲

**Comment by jakirkham** [comment]
Friendly nudge @r-jain1. Would you like a fix-up PR? It would be great to get this one in and will be much easier to maintain then too. 😄

**Comment by jakirkham** [comment]
This has been added via PR ( https://github.com/conda-forge/staged-recipes/pull/1252 ), which replaces this. Thanks for putting this together, @r-jain1, and everyone else for helping it along. 😄

**Pull request closed without merging**